docs/integrations/generic-commands-iam.md

```diff
@@ -0,0 +1,152 @@
+---
+id: generic-commands-iam
+title: Generic IAM Commands
+---
+
+
+## Background and motivation
+
+IAM integrations generally do not support CRUD (Create, Retrieve, Update, and Disable/Delete) operations for general user accounts or provisioning users. These integrations should adhere to a standard as the workflows can be generalized or completely customized to fit into an organization's environment.
+
+## Mappers
+In order to properly map between the generic IAM user fields and the data from the integration, we need to define an incoming [Mapper](../incidents/incident-classification-mapping#map-event-attributes-to-fields) 
```

Comment:
incoming for get-user, outgoing for update/create.
I think this is also the place to explain that their names should be the default values for "mapper-in" and "mapper-out" parameters.